guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -399,21 +399,27 @@ When you're happy with the code on your computer, you need to commit the changes
 $ git commit -a
 ```
 
-At this point, your editor should be fired up and you can write a message for this commit. Well formatted and descriptive commit messages are extremely helpful for the others, especially when figuring out why given change was made, so please take the time to write it.
+This should open a buffer in your editor for writing a commit message (when you
```

Comment:
"This should fire up your editor to write a commit message; when you have finished, save and close to continue."